rtos/rtx/TARGET_CORTEX_M/rt_CMSIS.c

```diff
@@ -821,6 +821,13 @@ os_InRegs osEvent_type svcThreadGetInfo (osThreadId thread_id, osThreadInfo info
     return osEvent_ret_value;
   }
 
+  if (osThreadInfoStackCurrent == info) {
+    uint8_t *stack_top = (uint8_t*)ptcb->stack + ptcb->priv_stack;
+    uint8_t *stack_ptr = (uint8_t*)ptcb->tsk_stack;
+    ret.value.v = stack_ptr - stack_top;
+    return osEvent_ret_value;
+  }
+
```

Comment:
I'm not sure I understand, is this not sufficient as a temporary implementation? All of these extensions will need to be updated with the new version of the RTOS.